stylesheet.css

```diff
@@ -0,0 +1,243 @@
+body {
+  background-color: #eff2f6;
+}
+.top-row .title {
+  font-family: "Inter", sans-serif;
+  margin: 0px 50px 0px 25px;
+}
+.top-row {
+  width: 100%;
+  height: 74px;
+  background-color: #ffffff;
+  display: flex;
+  margin: -6px 0 0 -6px;
+  padding: 18px 22.4px 16px;
+}
+.type-box {
+  border: 1px solid #aaa;
+  width: 203px;
+  height: 40px;
+  margin: 12px 15px 0 0;
+  float: left;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  justify-content: left;
+}
+.keywords {
+  font-size: 16px;
+  font-family: "Inter", sans-serif;
```

Comment:
jeśli chodzi o style to elementy dziedziczą po rodzicach. Tzn jeśli masz taką strukturę;
```
<div class="element-parent">
    <div class="element-child">AAA</div>
    <div class="element-child">BBB</div>
    <div class="element-other-child">CCC</div>
</div>
```
i chcesz nadać wszystkim elementom tą samą czcionkę to wystarczy że ustawisz ją dla elementu `element-parent`. Tak samo zadziała np. z wielkością czcionki. 

W Twoim przypadku można by dodać czcionkę i najbardziej powtarzającą się wielkość czcionki do `body`. Wtedy każdy nowy element będzie miał te wartości domyślne.